go.mod

```diff
@@ -232,3 +232,5 @@ replace gopkg.in/yaml.v2 => github.com/juju/yaml v0.0.0-20200420012109-12a32b78d
 replace github.com/dustin/go-humanize v1.0.0 => github.com/dustin/go-humanize v0.0.0-20141228071148-145fabdb1ab7
 
 replace github.com/hashicorp/raft-boltdb => github.com/juju/raft-boltdb v0.0.0-20200518034108-40b112c917c5
+
+replace github.com/juju/http/v2 => github.com/hmlanigan/http/v2 v2.0.0-20220518200255-65f466518f93
```

Comment:
Do we need this vs just updating juju/http/v2 ? Or maybe this is just to test the Juju level while waiting for your PR to land.

Reply:
this is temporary until the juju/http pr lands